### PR TITLE
Add a page for describing the history of forge.o.o.

### PR DESCRIPTION
**Pull request by gildor478**
Followup on the request from @avsm in #1172.

### CODE REVIEW

File: site/community/history/forge.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+with a common infrastructure. At this time, Github was not yet very popular and the other options did not offer the type
+of visibility for OCaml we wanted. When the forge was shutdown, there were 416 users and 300 projects hosted. 
+
+![Screenshot of the Forge in 2018](forge-2018.png)
+
+## Building the forge
+ 
+The main goal was to provide an infrastructure for the OCaml community:
+
+- Make more visible OCaml enthusiasts
```
Comment (dbuenzli) [inline site/community/history/forge.md]: Maybe rather 'Make OCaml enthusiasts more visible'

File: site/community/history/forge.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- Allow to gather an important code base
+- Centralize contribution to gain more momentum
```
Comment (dbuenzli) [inline site/community/history/forge.md]: contribution**s**

File: site/community/history/forge.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- Improve the OCaml “marketing” by showing many projects and libraries available
```
Comment (dbuenzli) [inline site/community/history/forge.md]: and the available libraries

File: site/community/history/forge.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  
+When the forge was built, the market was different. The main code hosting sites were SourceForge and Google Code. They
```
Comment (dbuenzli) [inline site/community/history/forge.md]: They were -> There were

File: site/community/history/forge.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+ 
+The main goal was to provide an infrastructure for the OCaml community:
+
+- Make more visible OCaml enthusiasts
+- Allow to gather an important code base
+- Centralize contribution to gain more momentum
+- Improve the OCaml “marketing” by showing many projects and libraries available
+  
+When the forge was built, the market was different. The main code hosting sites were SourceForge and Google Code. They 
+were other contenders like Savannah and self-hosting possibility with Trac. The team behind the forge has decided
```
Comment (dbuenzli) [inline site/community/history/forge.md]: has decided -> decided

**Review by dbuenzli — COMMENTED** [review]
I left what I believe are a few textual improvements. Thanks for the forge it was great to have that at some point.

**Comment by Chris00** [comment]
Thanks.

**Comment by agarwal** [comment]
@gildor478 Thank you for the summary. And thank you for running such a useful platform to the community for so many years!

**Pull request merged**